packages/edit-post/src/index.js

```diff
@@ -123,6 +123,12 @@ export function initializeEditor(
 	} );
```

Comment:
For completeness sake, should we add a `showListViewByDefault: false` line to the above `setDefaults` to match the one you've added in `edit-site`?

Reply:
Yes, totally!

I think I left it out because I wasn't sure if the key was the same in the preferences store (as opposed to the older features store), but it seems to have been synched.

Anyway, I'll add it. Good one!